dlls/gauss.cpp

```diff
@@ -314,12 +314,19 @@ void CGauss::StartFire( void )
 	
 	if ( gpGlobals->time - m_pPlayer->m_flStartCharge > GetFullChargeTime() )
 	{
+#ifdef CLIENT_DLL
+		flDamage = 200;
+#else
 		flDamage = gSkillData.plrDmgGaussSecondary;
+#endif
 	}
 	else
 	{
+#ifdef CLIENT_DLL
+		flDamage = 200 * (( gpGlobals->time - m_pPlayer->m_flStartCharge) / GetFullChargeTime() );
+#else
 		flDamage = gSkillData.plrDmgGaussSecondary * (( gpGlobals->time - m_pPlayer->m_flStartCharge) / GetFullChargeTime() );
-	}
+#endif
```

Comment:
I think a `}` was lost on the line just after `#endif`

Reply:
Yeah, I resolve this before you comment, but thanks anyway.